plat/kvm/Config.uk

```diff
@@ -10,7 +10,8 @@ menuconfig PLAT_KVM
        select ELF64_TO_32 if ARCH_X86_64
        select LIBUKRELOC if OPTIMIZE_PIE
        select LIBUKOFW if ARCH_ARM_64
-	   select LIBUKINTCTLR if ARCH_X86_64
+       imply LIBUKBUS_PCI if LIBUKBUS
+       select LIBUKINTCTLR if ARCH_X86_64
```

Comment:
Why `if ARCH_X86_64`?

Reply:
This is part of a different PR, I only fixed indentation here. But since you ask: Normally the platform would choose a specific driver (eg PIC, x2APIC, GICv2, GICv3), which in turn would select the API library, ie LIBUKINTCTLR. However, right now on x86 we don't support this yet, so we enable the library that provides the API. And on Arm we need to select the GIC driver at the VMM level, as QEMU virt by default uses GICv2 while FC uses GICv3.

Reply:
Ah, I see makes perfect sense, so this is temporary for x86 :+1: